pyccel/codegen/compiling/compilers.py

```diff
@@ -270,27 +269,30 @@ def _insert_prefix_to_list(lst, prefix):
 
     def _get_compile_components(self, compile_obj, accelerators = ()):
         """
-        Provide all components required for compiling
+        Provide all components required for compiling.
+
+        Provide all the different componenets (include directories, libraries, etc
+        which are needed in order to compile any file.
 
         Parameters
         ----------
-        compile_obj  : CompileObj
-                       Object containing all information about the object to be compiled
+        compile_obj : CompileObj
+            Object containing all information about the object to be compiled.
         accelerators : iterable of str
-                       Name of all tools used by the code which require additional flags/includes/etc
+            Name of all tools used by the code which require additional flags/includes/etc.
 
-        Results
+        Returns
         -------
-        exec_cmd      : str
-                        The command required to run the executable
-        inc_flags     : iterable of strs
-                        The include directories required to compile
-        libs_flags    : iterable of strs
-                        The libraries required to compile
-        libdirs_flags : iterable of strs
-                        The directories containing libraries required to compile
-        m_code        : iterable of strs
-                        The objects required to compile
+        str
+            The command required to run the executable.
+        iterable of strs
+            The include directories required to compile.
+        iterable of strs
+            The libraries required to compile.
+        iterable of strs
+            The directories containing libraries required to compile.
+        iterable of strs
+            The objects required to compile.
```

Comment:
Given that we have 5 return variables, I find it much easier to read this docstring (and navigate the code) if we keep the variable names. Did Codacy complain about it?

Reply:
No, numpydoc does

Reply:
Never mind. I misremembered. It seems fine.
Fixed 892a6ad